lib/texutil.js

```diff
@@ -59,8 +70,12 @@ try{ //try catch to make sure program still runs without csv file
         c++;
     }
     module.exports.paren = arr2set(paren);
-} catch (e) {}
-
+}
+catch (e) {
```

Comment:
I'm still not a fan of this catch expression.
Why not check with the fs library if the file exist?
Moreover, do we still ned the csv parser?
@HowardCohl Does it make sense to maintain two ways to define new macros?